sway/commands/output/allow_tearing.c

```diff
@@ -0,0 +1,22 @@
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "util.h"
+
+struct cmd_results *output_cmd_allow_tearing(int argc, char **argv) {
+	if (!config->handler_context.output_config) {
+		return cmd_results_new(CMD_FAILURE, "Missing output config");
+	}
+	if (argc == 0) {
+		return cmd_results_new(CMD_INVALID, "Missing allow_tearing argument");
+	}
+
+	if (parse_boolean(argv[0], true)) {
```

Comment:
I think we need to pass `config->handler_context.output_config->allow_tearing == 1` here instead of `true`, to make `toggle` work.

Reply:
Changed.